atomic_reactor/schemas/container.json

```diff
@@ -28,7 +28,34 @@
       "additionalProperties": false
     },
     "compose": {
-      "type": "object"
+      "type": "object",
+      "properties": {
+        "packages": {
+          "description": "names of RPMs to include",
+          "type": ["array", "null"],
+          "items": {
+              "type": "string"
+          }
+        },
+        "pulp_repos": {
+          "description": "whether to build pulp composes",
+          "type": "boolean"
+        },
+        "modules": {
+          "description": "names of modules to include",
+          "type": ["array", "null"],
+          "items": {
+              "type": "string"
+          }
+        },
+        "signing_intent": {
+          "type": ["string", "null"]
+        },
+        "with_multilib": {
+          "type": "boolean"
```

Comment:
Could add a description? e.g. "Whether to request multilib composes: a packages compose including x86_64 will also include i686; a pulp x86_64 compose will include i686"